src/tests/actions/blogActions.test.js

```diff
@@ -111,5 +127,34 @@ describe('blogActions', () => {
       expect(dispatchedActions.length).toBe(1);
       expect(dispatchedActions[0].type).toBe('ADD_NEW_COMMENT');
     });
+    test('checks if action.payload is correct and it has uuid', () => {
+      const store = mockStoreCreator({ entries: [] });
+      const newComment = { author: 'new author', text: 'new comment' };
+      const postUuid = '1';
+      store.dispatch(addNewComment(newComment, postUuid));
+      const dispatchedActions = store.getActions();
+      expect(dispatchedActions[0].payload).toMatchObject({
+        author: 'new author',
+        text: 'new comment',
+        uuid: expect.any(String)
+      });
+    });
+    test('checks if postUuid was successfully found', () => {
```

Comment:
I think this test should be combined with the above and include the assertion on `postUuid`.